charts/public-service/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "service.fullname" . }}
+  labels:
+    app: {{ template "service.name" . }}
+    chart: {{ template "service.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  revisionHistoryLimit: 2
```

Comment:
why is the `revisionHistoryLimit` non-configurable?

Reply:
we might simply remove it. the default value is `10` which is totally fine (https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy)

Reply:
8923513